src/policy.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// NOTE: This file is intended to be customised by the end user, and includes only local node policy logic
+
+#include "policy.h"
+
+#include "amount.h"
+#include "tinyformat.h"
+#include "ui_interface.h"
+#include "utilstrencodings.h"
+
+bool fIsBareMultisigStd = true;
+/** Fees smaller than this (in satoshi) are considered zero fee (for relaying and mining) */
+CFeeRate minRelayTxFee = CFeeRate(1000);
+static const unsigned int MAX_OP_RETURN_RELAY = 40; //! bytes
+
+/** Declaration of Standard Policy implementing CPolicy */
+class CStandardPolicy : public CPolicy
+{
+protected:
+    unsigned nMaxDatacarrierBytes;
+public:
+    CStandardPolicy() : nMaxDatacarrierBytes(MAX_OP_RETURN_RELAY) {};
+
+    virtual void InitFromArgs(const std::map<std::string, std::string>&);
+    virtual bool ValidateScript(const CScript&, txnouttype&) const;
+};
+
+/** Global variables and their interfaces */
+
+static CStandardPolicy standardPolicy;
+
+static CPolicy* pCurrentPolicy = 0;
+
+CPolicy& Policy(std::string policy)
+{
+    if (policy == "standard")
+        return standardPolicy;
+    throw std::runtime_error(strprintf(_("Unknown policy '%s'"), policy));
+}
+
+void SelectPolicy(std::string policy)
+{
+    pCurrentPolicy = &Policy(policy);
+}
+
+const CPolicy& Policy()
+{
+    assert(pCurrentPolicy);
+    return *pCurrentPolicy;
+}
+
+std::string GetPolicyUsageStr()
```

Comment:
Mhmm, so move -policy back to init and then have this function take a string parameter so that it can return different things for different policies?. Is that what you're suggesting?